src/Controller/ScheduledTaskController.php

```diff
@@ -65,6 +65,24 @@ public function runTask(string $id, Context $context): JsonResponse
         return new JsonResponse(null, JsonResponse::HTTP_NO_CONTENT);
     }
 
+    /**
+     * @Route(path="/scheduled-task/inactive/{id}", methods={"POST"}, name="api.frosh.tools.scheduled.task.inactive")
+     */
+    public function setTaskInactive(string $id, Context $context): JsonResponse
+    {
+        $scheduledTask = $this->fetchTask($id, $context);
+
+        // Set status to allow running it
+        $this->scheduledTaskRepository->update([
+            [
+                'id' => $id,
+                'status' => ScheduledTaskDefinition::STATUS_INACTIVE,
```

Comment:
Can you build it toggleable? When you disable it now with the UI, you can't enable it again

Reply:
Sadly I won't have time till before the first of june. Happy to extend it later on. I think it would come in handy anyways, but up to you